internal/kms/kms.go

```diff
@@ -103,6 +103,11 @@ func (k *Kms) AddExternalWrappers(ctx context.Context, opt ...Option) error {
 			return errors.Wrap(ctx, err, op, errors.WithMsg("unable to add worker auth wrapper"))
 		}
 	}
+	if opts.withWorkerStorageWrapper != nil {
```

Comment:
WorkerStorage also needs to be added to the `GetExternalWrappers` method in this file

Can you also add the WorkerStorage wrapper to these files:
`globals/kms.go` 
`internal/kms/external_wrappers.go`
`internal/kms/external_wrappers_test.go`